### PR TITLE
qtwebengine: fix default screen information retrieval

### PR DESCRIPTION
**Pull request by Tofee**
Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>

### CODE REVIEW

**Comment by shr-project** [comment]
Did you forget to add the .patch file or is it just missing in SRC_URI?

**Comment by Tofee** [comment]
Fixed. Thanks for your sharp eyes !

File: meta-luneui/recipes-qt/qt5/qtwebengine/0012-screenInfoFromQScreen-fallback-on-primary-screen-if-.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+diff --git a/src/core/render_widget_host_view_qt.cpp b/src/core/render_widget_host_view_qt.cpp
+index 3be0c939..a37f9940 100644
+--- a/src/core/render_widget_host_view_qt.cpp
++++ b/src/core/render_widget_host_view_qt.cpp
+@@ -255,6 +255,9 @@ private:
+ static content::ScreenInfo screenInfoFromQScreen(QScreen *screen)
+ {
+     content::ScreenInfo r;
++    
++    if (!screen) screen = qGuiApp->primaryScreen();
```
Comment (Herrie82) [inline meta-luneui/recipes-qt/qt5/qtwebengine/0012-screenInfoFromQScreen-fallback-on-primary-screen-if-.patch]: Would be nice to have the screen = qGuiApp->primaryScreen(); inside of {} like the other if conditions.

Comment (Tofee) [reply meta-luneui/recipes-qt/qt5/qtwebengine/0012-screenInfoFromQScreen-fallback-on-primary-screen-if-.patch]: For one-liners, the rest of the code uses just a indent, no {}.
I can do the change, but I'm not sure the patch would be more clear?

Comment (Herrie82) [reply meta-luneui/recipes-qt/qt5/qtwebengine/0012-screenInfoFromQScreen-fallback-on-primary-screen-if-.patch]: I was just thought to always use {} after an if for clarity ;)

**Pull request merged**